tests/metadata_test.ml

```diff
@@ -155,13 +144,13 @@ let capture_article_metadata_invalid3 =
     Preface.Pair.fst
     $ split_metadata {|---
 ---My article|}
-    |> Metadata.Article.from_string
+    |> Metadata.Article.from_string (module Yocaml_yaml)
   in
   check
   $ opt_validate_testable Format.pp_print_string String.equal
   $ "should be equal"
   $ Validate.Monad.(obj >|= Metadata.Article.title)
-  $ Error.(to_validate $ Invalid_metadata "Article")
+  $ Error.(to_validate $ Invalid_field "Object expected")
```

Comment:
Ah ok, you have modified here, I understand better :) (in reference to the test launch). Your choice seems to be explicit!